<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>preferencesForm</class>
 <widget class="QMainWindow" name="preferencesForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>resources/icon.png</normaloff>resources/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <widget class="QWidget" name="thumbnailsTab">
       <attribute name="title">
        <string>Image</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QWidget" name="sizeWidget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="sizeLabel">
             <property name="text">
              <string>Thumbnail size, px (the biggest side)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sizeSpinBox"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="infoBox">
          <property name="title">
           <string>Image Info</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QCheckBox" name="similarityBox">
             <property name="text">
              <string>Show similarity rate</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="sizeBox">
             <property name="text">
              <string>Show image size</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="pathBox">
             <property name="text">
              <string>Show image path</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="sizeFormatWidget" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="sizeFormatLabel">
                <property name="text">
                 <string>Image size format</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="sizeFormatComboBox"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="sortWidget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item alignment="Qt::AlignLeft">
            <widget class="QLabel" name="sortLabel">
             <property name="text">
              <string>Sort images by</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="sortComboBox">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="currentText">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="searchTab">
       <attribute name="title">
        <string>Search</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item alignment="Qt::AlignTop">
         <widget class="QWidget" name="widget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QGroupBox" name="dimsBox_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Image Size (px)</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QWidget" name="widget_7" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_8">
                 <item>
                  <widget class="QWidget" name="widget_8" native="true">
                   <layout class="QHBoxLayout" name="horizontalLayout_11">
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QLabel" name="minWLabel_2">
                      <property name="text">
                       <string>Min Width</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSpinBox" name="minWSpinBox_2"/>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QWidget" name="widget_9" native="true">
                   <layout class="QHBoxLayout" name="horizontalLayout_12">
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QLabel" name="minHLabel_2">
                      <property name="text">
                       <string>Min Height</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSpinBox" name="minHSpinBox_2"/>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget_10" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_9">
                 <item>
                  <widget class="QWidget" name="widget_11" native="true">
                   <layout class="QHBoxLayout" name="horizontalLayout_13">
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QLabel" name="maxWLabel_3">
                      <property name="text">
                       <string>Max Width</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSpinBox" name="maxWSpinBox_3"/>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QWidget" name="widget_12" native="true">
                   <layout class="QHBoxLayout" name="horizontalLayout_14">
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QLabel" name="maxHLabel_3">
                      <property name="text">
                       <string>Max Height</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSpinBox" name="maxHSpinBox_3"/>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="subfoldersBox">
             <property name="text">
              <string>Recursive search (include subfolders)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="betweenDirsBox_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Only between chosen folders</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="miscTab">
       <attribute name="title">
        <string>Misc</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item alignment="Qt::AlignTop">
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QCheckBox" name="deldirsBox">
             <property name="text">
              <string>Delete folders if they are empty after deleting/moving images</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="closeBox">
             <property name="text">
              <string>Ask confirmation when a user closes the programme</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item alignment="Qt::AlignRight">
     <widget class="QWidget" name="buttonsWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="saveBtn">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelBtn">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
